GoRogue/MathHelpers.cs

```diff
@@ -91,5 +91,18 @@ public static double ScaledAtan2Approx(double y, double x)
                 return x < 0.0 ? y < 0.0 ? 0.5 + r : 0.5 - r : y < 0.0 ? 1.0 - r : r;
             }
         }
+
+        /// <summary>
```

Comment:
This is a duplicate of `Distance.Euclidean.Calculate(Point, Point)` and should be replaced with that.